apps/ledger-live-desktop/src/renderer/families/celo/ActivateFlowModal/steps/StepConfirmation.jsx

```diff
@@ -27,6 +27,7 @@ export const StepConfirmationFooter = ({
 }: StepProps) => {
   return (
     <Box horizontal alignItems="right">
+      <SyncOneAccountOnMount priority={10} accountId={account.id} />
       <Button data-test-id="modal-close-button" ml={2} onClick={onClose}>
```

Comment:
For all staking confirmation steps, I'm now rendering the `<SyncOneAccountOnMount />` component in the StepConfirmationFooter to ensure that it will always be rendered.  

Previously, it was only rendered when an optimistic operation existed - which may or may not exist.

Reply:
Because this is different from the usual pattern in other integrations, could you please add a comment explaining basically what you just said + a quick bit of context about the issue it's trying to solve? :pray: thanks.

Reply:
@haammar-ledger - done!